CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+## v7.1.0
+
+Infrastructure changes:
+
+- `@mapbox/maki` can now be used from ESM modules and browser.
```

Comment:
Have you documented this usage anywhere? It might be nice to include an example of doing this for ESM and Common JS in the readme?